docs/tutorials/lander.md

```diff
@@ -0,0 +1,376 @@
+    `)
+})
+```
+
+## Step 13 @fullscreen
+
+Right click on the ``||controller:on left button pressed||`` and duplicate it twice. Change the buttons to ``right`` and ``up``. For ``right`` change the text to ``"right"`` and for ``up`` change the text to ``"straight"``. Finally, draw an image of a spacecraft going diagonally right-up for ``right``. Copy the first spacecraft drawing for straight and paste it into the image editor for the ``up`` button.
+
+```blocks
+let lander: Sprite = null
+controller.left.onEvent(ControllerButtonEvent.Pressed, function () {
+    angle = "left"
+    lander.setImage(img`
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . d d d . . . . . . . .
+        . . . . d d d d d . . . . . . .
+        . . . d d d 1 1 d d . . . . . .
+        . . . d d 1 1 1 d d . . . . . .
+        . . . d d 1 1 d d d f f . . . .
+        . . . . d d d d d . . . f . . .
+        . . . . . d d d . . . . . 2 . .
+        . . . . . . . f . . . . . . . .
+        . . . . . . . f . . . . . . . .
+        . . . . . . . . f . . . . . . .
+        . . . . . . . . . 2 . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+    `)
+})
+controller.right.onEvent(ControllerButtonEvent.Pressed, function () {
+    angle = "right"
+    lander.setImage(img`
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . d d d . . . . .
+        . . . . . . . d d d d d . . . .
+        . . . . . . d d 1 1 d d d . . .
+        . . . . . . d d 1 1 1 d d . . .
+        . . . . f f d d d 1 1 d d . . .
+        . . . f . . . d d d d d . . . .
+        . . 2 . . . . . d d d . . . . .
+        . . . . . . . . f . . . . . . .
+        . . . . . . . . f . . . . . . .
+        . . . . . . . f . . . . . . . .
+        . . . . . . 2 . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+    `)
+})
+controller.up.onEvent(ControllerButtonEvent.Pressed, function () {
+    angle = "straight"
+    lander.setImage(img`
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . d d d . . . . . . .
+        . . . . . d d d d d . . . . . .
+        . . . . d d 1 1 1 d d . . . . .
+        . . . . d d 1 1 1 d d . . . . .
+        . . . . d d d d d d d . . . . .
+        . . . . . d d d d d . . . . . .
+        . . . . . f d d d f . . . . . .
+        . . . . f f . . . f f . . . . .
+        . . . 2 . . . . . . . 2 . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+        . . . . . . . . . . . . . . . .
+    `)
+})
+```
+
+## Complete @fullscreen
+
+Congratulations, you have completed your game! Now try and make your own levels!
```

Comment:
What are levels? Difficulty levels, achievement levels, or new platforming?

Reply:
Levels is meant to be the tile maps. They can create games with different scenes/complexity through the tile map. What should I rename it to?

Reply:
maybe just 'more' or 'other' levels instead of 'your own' levels, as the level they created is already their own level already?

Reply:
Okay, I'll make the change and push it.

Reply:
"Add more tile maps and create other levels of play."... or some such.